packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -553,8 +556,7 @@ function _DataViewsViewConfig( {
 		<>
 			<ViewTypeMenu defaultLayouts={ defaultLayouts } />
 			<Dropdown
-				popoverProps={ { placement: 'bottom-end', offset: 9 } }
-				contentClassName="dataviews-view-config"
+				popoverProps={ DATAVIEWS_CONFIG_POPOVER_PROPS }
 				renderToggle={ ( { onToggle } ) => {
 					return (
 						<Button
```

Comment:
While you’re at it `aria-expanded` would be nice.

```suggestion
				renderToggle={ ( { onToggle, isOpen } ) => {
					return (
						<Button
							aria-expanded={ isOpen }
```

Reply:
Added `aria-expended` and `aria-controls`